src/database/queries/getPosts.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../connection.js');
+const getPosts = () => connection.query(`select * from posts`);
+module.exports = getPosts;
```

Comment:
@denaHS Very minor thing: It is sometimes helpful to name these types of queries (for example) `getAllPosts`, so that other people on the team know that if they call this function, they will get all of the posts in the database (with no limits, potentially a lot of posts).